src/core/Akka/Actor/Settings.cs

```diff
@@ -136,6 +148,9 @@ public Settings(ActorSystem system, Config config, ActorSystemSetup setup)
                 }
             }
             
+            // set the filter
+            StdoutLogger!.Filter = LogFilter;
```

Comment:
New property added to the `MinimaLogger` - it's settable `internal`ly so we do that here. This is what ties the `LogFilterEvaluator` into the `StandardOutLogger` in default Akka.NET logging scenarios.